modules/virt-change-vm-instance-type.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-change-vm-instance-type_{context}"]
+
+= Changing a VM's instance type
```

Comment:
```suggestion
= Changing an instance type of a VM
```

https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> Do not use possessive ’s with inanimate objects.

Reply:
I have no idea where I got the idea that this was ok 😂